README.md

```diff
@@ -0,0 +1,72 @@
+# IHP Boilerplate
+
+This repository contains a boilerplate for IHP (Integrated Haskell Platform) projects with GitHub Actions for testing and deployment, used by `ihp-new`, see the [Installation Guide](https://ihp.digitallyinduced.com/Guide/installation.html).
```

Comment:
The readme will be copied over into any new project created with ihp-new. So it would be better to have this wording from the perspective of "this is an IHP project" instead of referring to the ihp boilerplate